test/e2e/v3/generate.go

```diff
@@ -0,0 +1,219 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v3
```

Comment:
By following the convection test files should be named with `_test.go`. Could you please change that? also, see that coveralls ignore the _test.go by default as well. 

PS.: we need cleanup the tests, however, it is fine we can do it a follow up.